tools/vipps-partner-api-postman-environment.json

```diff
@@ -35,7 +35,31 @@
 		},
 		{
 			"key": "access_token",
-			"value": "<This value will be overwritten when you call the Get Access Token API>",
+			"value": "<This value will be overwritten when you call the Get Access Token API when having selected eCommerce v2 - MTE environment>",
```

Comment:
They shouldn't need to change environment to call the Get Access Token API.

Reply:
I will revert this back to the previous definition.